src/span.h

```diff
@@ -107,7 +107,7 @@ class Span
 
 
 public:
-    constexpr Span() noexcept : m_data(nullptr), m_size(0) {}
+    constexpr Span() noexcept : m_data(nullptr) {}
```

Comment:
Any reason why nullptr isn't fixed as well?

Reply:
Yeah, it has been overlooked. The `clang-tidy` does not complain, though.

Reply:
I guess it refrains because there are multiple constructors: https://github.com/llvm/llvm-project/blob/78f88082de3627ea04501c83a08f52cf1e60b4f7/clang-tools-extra/test/clang-tidy/checkers/modernize/use-default-member-init.cpp#L56